src/roman_datamodels/stnode.py

```diff
@@ -224,7 +240,7 @@ class LNode(UserList):
     def __init__(self, node=None):
         if node is None:
             self.data = []
-        elif isinstance(node, list):
+        elif isinstance(node, Iterable):
```

Comment:
```suggestion
        elif isinstance(node, list):
```
I think  this can also be reverted.